modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/SeriesRestService.java

```diff
@@ -345,6 +350,91 @@ public Response getSeriesAccessControlListJson(@PathParam("seriesID") String ser
     return getSeriesAccessControlList(seriesID);
   }
 
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/allInRangeAdministrative.json")
+  @RestQuery(
+      name = "allInRangeAdministrative",
+      description = "Internal API! Returns all series (included deleted ones!) in the given "
+          + "range 'from' (inclusive) .. 'to' (exclusive). Returns at most 'limit' many series. "
+          + "Can only be used as administrator!",
+      returnDescription = "Series in the range",
+      restParameters = {
+          @RestParameter(
+              name = "from",
+              isRequired = true,
+              description = "Start of date range (inclusive) in milliseconds "
+                  + "since 1970-01-01T00:00:00Z. Has to be >=0.",
+              type = Type.INTEGER
+          ),
+          @RestParameter(
+              name = "to",
+              isRequired = false,
+              // TODO: this shows the default value as 0 despite us not setting this value!
```

Comment:
```suggestion
              defaultValue = "",
```

Reply:
Nop, that still shows `0` in the test form box :/

Reply:
Interesting. Well… let's ignore that for now.